tests/Signature/SignatureV4Test.php

```diff
@@ -277,6 +277,32 @@ public function testProvider()
     /**
      * @dataProvider testProvider
      */
+    public function testSignRequestUnsignedPayload($req, $sreq, $creq)
```

Comment:
I am going to do this with a new provider since all signatures will need to be changed as well.